index.js

```diff
@@ -81,7 +81,8 @@ testRunner([
 			return validator;
 		},
 		test: function (instance, json, schema) {
-			return instance.validate(json, schema).valid;
+			// third argument is `breakOnError` to halt validation at first error
+			return instance.validate(json, schema, true).valid;
```

Comment:
I do not want to start adding special settings for each validator. The tests should run it in the complete standard way, that most users would use if they are just picking up the validator and start using it.